app/ColorPair.js

```diff
@@ -0,0 +1,9 @@
+module.exports = class ColorPair {
+    constructor() {
+        this.majorColor;
+        this.minorColor;
+    }
+    toString() {
+        return `MajorColor:${this.majorColor},MinorColr:${this.minorColor}`;

```

Comment:
Note the spelling: MinorColr. Write a test to catch this and then fix it. Consider also, if a user would like to see 'MajorColor' or 'Major Color'